data/provider/src/main/java/tech/pegasys/teku/api/GetSpecResponse.java

```diff
@@ -36,25 +36,8 @@ public Map<String, String> getConfigMap() {
         .getRawConfig()
         .forEach((name, value) -> configAttributes.put(name, ConfigProvider.formatValue(value)));
 
-    configAttributes.put("GOSSIP_MAX_SIZE", getGossipMaxSize());
```

Comment:
i'm concerned about removing these.

If they're in the YAML that we just got for holesky, why would we remove them? (only checked first one)

Reply:
all atributes already arrives from rawConfig except those from hardcoded constants

Reply:
we will not start with config without these attributes